plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-notices.shopper.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * External dependencies
+ */
+import { expect, test as base } from '@woocommerce/e2e-playwright-utils';
+import {
+	cli,
+	BLOCK_THEME_SLUG,
+	BLOCK_CHILD_THEME_SLUG,
+} from '@woocommerce/e2e-utils';
+/**
+ * Internal dependencies
+ */
+import { CheckoutPage } from '../checkout/checkout.page';
+import { REGULAR_PRICED_PRODUCT_NAME } from '../checkout/constants';
+
+const test = base.extend< { checkoutPageObject: CheckoutPage } >( {
+	checkoutPageObject: async ( { page }, use ) => {
+		const pageObject = new CheckoutPage( {
+			page,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( 'Shopper → Block Notice Templates', () => {
+	test.beforeEach( async ( { wpCliUtils, frontendUtils } ) => {
```

Comment:
Should we be setting the cart page like this on every test, can we just run it once before the suite? (same for unsetting it in the `afterEach`).

I think we should move those sections to a `before` and `after` block, but leave the code that needs to run before/afer _each test_ in `beforeEach` and `afterEach`

Reply:
Initially, I was using `beforeAll` and `afterAll` instead of `beforeEach` and `afterEach`, but I ended up with an error message due to page scopes. 

> I think we should move those sections to a before and after block, but leave the code that needs to run before/afer each test in beforeEach and afterEach

You mean moving the CLI related calls into `beforeAll` and keep the `frontendUtils` related parts in `beforeEach`?

Reply:
Yep!